### PR TITLE
[CI] Remove references to ENV vars in Podfile / circle.yml

### PR DESCRIPTION
**Pull request by orta**
Betas would get OSS keys, they would work... Betas get deployed to the app store. They don't work for a reviewer. Phew.

This also adds a check that we're not deploying those keys through fast lane.

Richer [context in slack](https://artsy.slack.com/archives/eigen/p1452257377000467)

### CODE REVIEW

**Comment by orta** [comment]
I've set all the ENV vars in the admin - https://circleci.com/gh/artsy/eigen/edit#env-vars

**Comment by orta** [comment]
Build fail is Danger. I will look at this.

**Comment by ashfurrow** [comment]
Looks :+1: to me.

**Comment by orta** [comment]
OK, I might be good with this. If it greens.

**Comment by orta** [comment]
@alloy - this PR introduces the full dependency of `fastlane`. I wanted to be able to bring eigen more in line with the other libs, and making sure that we're submitting to (dsyms to) hockey, (apps to) testflight and then letting us know via slack

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Error
    ------------- | ------------
        :no_entry_sign: | No CHANGELOG changes made
    

  

  
    &nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

File: fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+  mini_readme = readme.split("\n## ")[0..1].join("\n## ")
+  generated_changelog = beta_readme + "\n\n-------\n" + mini_readme
+
+  # Build
+  gym
+
+  # Ship to hockey first, testflight has to wait: "processing"
+  hockey notes: beta_readme, api_token: ENV['HOCKEY_API_TOKEN']
+  pilot changelog: readme
```
Comment (alloy) [inline fastlane/Fastfile]: So beta builds go to both?

Comment (orta) [inline fastlane/Fastfile]: Yes, this sends the dsym, and when you migrate a beta -> production, then it'll keep the dsym

**Comment by alloy** [comment]
I’m fine with all the dependencies if we’re actually using it all :+1:

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Comment by orta** [comment]
OK, merging on :green_apple:

**Comment by ArtsyMobile** [comment]
&nbsp; | 1 Warning
    ------------- | ------------
        :warning: | Needs testing on a Phone if change is non-trivial
    

  

  


<p align="right" meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://github.com/KrauseFx/danger/">danger</a>
</p>

**Pull request merged**